pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "yapapi"
-version = "0.3.0-alpha.2"
+version = "0.3.0-alpha.3"
```

Comment:
@mfranciszkiewicz cool, though the version updates should only be made when we're actually building a version